docs/user-guide/release-notes/2.0.x.rst

```diff
@@ -1,6 +1,19 @@
 2.0 Release Notes
 =================
 
+2.0.1
```

Comment:
Indeed!

That PR is merged, so a quick rebase onto the new 2.0-dev is in order. Comically, if we're building another beta, I think this will likely get pulled into 2.0.1.